React/Views/RCTWebViewManager.m

```diff
@@ -79,4 +79,16 @@ - (NSDictionary *)constantsToExport
   }];
 }
 
+RCT_EXPORT_METHOD(evaluateJavaScript:(NSNumber *)reactTag script: (NSString *)script callback:(RCTResponseSenderBlock) callback)
+{
+  [self.bridge.uiManager addUIBlock:^(RCTUIManager *uiManager, RCTSparseArray *viewRegistry) {
+    RCTWebView *view = viewRegistry[reactTag];
+    if (![view isKindOfClass:[RCTWebView class]]) {
+      callback(@[[NSString stringWithFormat:@"Invalid view returned from registry, expecting RKWebView, got: %@", view], [NSNull null]]);
```

Comment:
This should use `RCTLogError` (which causes a redbox) like the other methods in this file.